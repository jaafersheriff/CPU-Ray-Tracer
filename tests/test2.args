pixelray simple.pov 640 480 170 120
