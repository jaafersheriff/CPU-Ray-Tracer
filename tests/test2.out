Pixel: [170, 120] Ray: {0 0 14} -> {-0.2893 -0.2313 -0.9289}
